pkg/config/config.go

```diff
@@ -167,6 +177,14 @@ func FromMap(confMap map[string]any) (Config, error) {
 	return conf, nil
 }
 
+func (config *Config) SetDefaults() {
```

Comment:
Can't we set this in the the default configuration instead? 
https://github.com/StyraInc/regal/blob/main/bundle/regal/config/provided/data.yaml

Reply:
Not trivial, but this is the right way you're right...

72dcabf6d09415a7adee1bd7759cb4f099578222 is the commit that adjusts this PR to do that, lmk if you think that looks ok.